src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-do List</title>
+</head>
+
+<body>
+    <main>
+        <section class="to-do-list">
+            <h1> To-Do List</h1>
+            <div class="list-container">
```

Comment:
The only thing that I could recommend is to better use the HTML 5 semantic tags, as normally you would have a main if there are any other sections in the code that you want to be secondary (header, footer, etc).